pyproject.toml

```diff
@@ -31,17 +32,24 @@ include = [
 "Bug Tracker" = "https://github.com/nautobot/nautobot-app-golden-config/issues"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
+python = ">=3.8,<3.13"
 deepdiff = ">=5.5.0,!=6.0,!=6.1,<8"
 django-pivot = ">=1.9.0,<1.10.0" # The signature changed to return a non-queryset, do not upgrade without ensuring it returns a queryset
 matplotlib = "^3.3.2"
 nautobot-plugin-nornir = "^2.0.0"
+# Numpy is not a direct dependency, but we are assisting Poetry version resolution by providing constraints.
+numpy = [
+    {version = "^1.26", python = ">=3.9"},
+    {version = "<1.25", python = "==3.8"}
+]
 
 toml = "^0.10.2"
 netutils = "^1.5.0"
 hier-config = "^2.2.2"
 nautobot-capacity-metrics = "^3.0.1"
 xmldiff = "^2.6.3"
+# Used for local development
+nautobot = "^2.0.0"
```

Comment:
We have avoided doing this in the past as it requires 2 updates with nautobot-plugin-nornir as well.

Reply:
Can you reference the 2 updates that would need to be done to nautobot-plugin-nornir?

Reply:
This line would be 1, the other would be https://github.com/nautobot/nautobot-app-nornir/blob/af7ba4d4e5dfa538875d9e4bde1aaf9f8d4fa06a/pyproject.toml#L37